sw/device/tests/sim_dv/sysrst_ctrl_ec_rst_l_test.c

```diff
@@ -0,0 +1,222 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/dif/dif_sysrst_ctrl.h"
+#include "sw/device/lib/runtime/ibex.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/rstmgr_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+static dif_pinmux_t pinmux;
+static dif_sysrst_ctrl_t sysrst_ctrl;
+static dif_pwrmgr_t pwrmgr;
+static dif_rstmgr_t rstmgr;
+static dif_rstmgr_reset_info_bitfield_t rstmgr_reset_info;
+
+const uint32_t kTestPhaseTimeoutUsec = 2500;
+
+enum {
+  kTestPhaseSetup = 0,
+  kTestPhaseCheckComboReset = 1,
+  kTestPhaseOverrideSetup = 2,
+  kTestPhaseOverrideZeros = 3,
+  kTestPhaseOverrideOnes = 4,
+  kTestPhaseDone = 5,
+};
```

Comment:
Nit: This might be clearer if you defined and then used an enumerated type for test phases.

Reply:
I am not clear.. Are these enum phases are used below? Or do you suggest the same names in both vseq and .c files? Please let me know

Reply:
Below you're assigning these enums to a uint8_t when you could use a typedef to make your own test_phase_t to make things a little clearer.

```
typedef enum test_phase {
...
} test_phase_t;
```

Reply:
Thanks. It's updated.